src/StructuredLogViewer/app.manifest

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+  <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>

```

Comment:
what should the identity be? Perhaps MSBuildStructuredLogViewer.app?

Reply:
if this is intended to be the assembly name, then perhaps StructuredLogViewer.app?

Reply:
The article says:
> Uniquely names the application or assembly. Use the following format for the name: `Organization.Division.Name`. For example `Microsoft.Windows.mysampleApp`. Required.

Maybe we can do something like `MSBuildStructuredLog.StrucutredLogViewer`/`MSBuildStructuredLog.TaskRunner`? These names aren't used for anything, we don't use the unmanaged assembly binding, it's just for tidiness